src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -887,6 +888,7 @@ private void HeaderReadComplete(IAsyncResult result)
                 }
 
                 int packetLength = BitConverter.ToInt32(_headerByte, 1);
+                MSBuildEventSource.Log.PacketReadSize(packetLength);
 
                 byte[] packetData;
                 if (packetLength < _smallReadBuffer.Length)
```

Comment:
nit/pre-existing: This should be `<=` as both are lengths so no need to subtract one.